docs/user_guide/event_system.md

```diff
@@ -0,0 +1,147 @@
+
+| Type | Event | Notes |
+|--|--|--|
+| Application | A new application is created ||
+| Application | Application state transition ||
+| Application | Successful allocation ||
+| Application | An ask is added to the application ||
+| Application | Application removed ||
+| Application | Application rejected ||
+| Application | Ask removed ||
+| Application | Allocation removed ||
+| Application | Placeholder timeout ||
+| Application | Allocation replacement (gang scheduling) ||
+| Application | Application is not runnable due to maxApplications | Queue or user maxApps limit |
+| Application | Application has become runnable | Only sent if limit was hit before |
+| Application | Ask is not schedulable due to quota limitations | Sent to Kubernetes to the respective pod |
+| Application | Ask has become schedulable (there is quota now) | Sent to Kubernetes to the respective pod<br/> Only sent if quota was not available before |
+| Application | Ask is not schedulable due to predicate error * | Sent to Kubernetes to the respective pod<br/> Rate limited: sent once in every 15 seconds |
+| Application | Real allocation is larger than the placeholder | Sent to Kubernetes to the respective pod |
+| Node | New node added to the scheduler core | Sent to Kubernetes to the respective node |                             
+| Node | Node removed from the scheduler core | Sent to Kubernetes to the respective node |
+| Node | Allocation added to a node ||
+| Node | Allocation removed from a node ||
+| Node | Node "ready" status changed ||
+| Node | Node "schedulable" status changed ||
+| Queue | Queue added to the scheduler core ||
+| Queue | Application added to a queue ||
+| Queue | Queue removed from the scheduler core ||
+| Queue | Max resource changed | From leaf to parent or vice-versa ||
+| Queue | Queue type changed ||
+| User/group | Limit configured for a user ||
+| User/group | Limit configured for a group ||
+| User/group | Limit removed for a user ||
+| User/group | Limit removed for a group ||
+| User/group | Resource usage increased for a user ||
+| User/group | Resource usage increased for a group | Only sent if group limit is active |
+| User/group | Group tracking got associated with a user| Only sent if group limit is active |
+| User/group | Group tracking removed from a user| Only sent if group limit is active |
+
+`*` a predicate is a plugin located in the default scheduler. They're responsible for functionality like node selectors, affinity, anti-affinity, etc. Yunikorn runs the predicates for every allocation to see if it fits the candidate node.
+
+# In-memory storage of historical events
+
+Yunikorn stores a previously generated event objects in a ring buffer. Once it gets full, oldest elements are overwritten. The default size of the buffer is 100000.
+On a busy cluster this might not be enough. The size can be changed in the configmap by using the key `event.ringBufferCapacity`.
```

Comment:
I didn't see below configs in  [YuniKorn Configuration](https://yunikorn.apache.org/docs/next/user_guide/service_config#yunikorn-configuration) page

- event.ringBufferCapacity
- event.trackingEnabled

Is it missing?
Is '[service.eventChannelCapacity](https://yunikorn.apache.org/docs/next/user_guide/service_config#serviceeventchannelcapacity)' equal to 'event.ringBufferCapacity'?

Reply:
Actually, four properties are missing.

`event.requestCapacity` is completely ignored now, it should be fixed.

Reply:
@chia7712 The missing configs are fixed in the latest commit, so I think https://issues.apache.org/jira/browse/YUNIKORN-2455 could be closed.

Reply:
> The missing configs (4 of 5) are fixed in the latest commit, so I think https://issues.apache.org/jira/browse/YUNIKORN-2455 could be closed.

done